contributing/content-style-guide.md

```diff
@@ -440,13 +440,15 @@ username | login
 
 ## Links
 
-Links embedded in documentation are about trust. Users trust that we will lead them to sites or pages related to their reading content. In order to maintain that trust, it is important that links are transparent, up-to-date, and lead to legitimate resources.
+Links embedded in documentation are about trust. Users trust that we will lead them to sites or pages related to their reading content. In order to maintain that trust, it's important that links are transparent, up-to-date, and lead to legitimate resources.
 
 ### Internal links
 
-The full URL is not required when linking to another page in the docs.getdbt.com domain. This is preferred since the Docusaurus markdown will open the link in the same window rather than creating a new browser tab. You can use the path to the page (everything in the URL bar after docs.getdbt.com).
+We require root-relative _file_ paths (include the file extension, such as `.md`) or root-relative _URL_ paths (don't include `.md`) without the domain, docs.getdbt.com. We prefer this over using document-relative paths (relating to your current document, like one directory above it `../LinkedDocument`) because root-relative paths will work properly during local development and testing, and moving a document won't break the links it contains.  
```

Comment:
```suggestion
All internal links should use relative and not absolute paths. We construct these paths in relation to the content root, which is `[https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs)`, and they don't include the domain, https://docs.getdbt.com.

We require  either _file_ paths relative to the content root (these include the file extension, such as `.md`) or _URL_ paths relative to the content root (these don't include `.md`). We avoid paths relative to the document (for example, one directory above a document `../LinkedDocument`) because they won't work during local development and testing, and moving a document won't break the links it contains.  
```